ETABS_Engine/Convert/ToCSI.cs

```diff
@@ -85,6 +85,25 @@ public static void ToCSI(this BarRelease release, ref bool[] startRestraint, ref
             endSpring[3] = release.EndRelease.RotationalStiffnessX;
             endSpring[4] = release.EndRelease.RotationalStiffnessY;
             endSpring[5] = release.EndRelease.RotationalStiffnessZ;
+
+            bool[] startReleased = startRestraint.Zip(startSpring, (x, y) => x && y == 0).ToArray();
```

Comment:
Should take this opportunity to double check the direction of the releases here. Especially Uy,Uz,Ry and Rz and make sure the convert from BHoM to Etabs 1-3-2 system is done correctly.